messages/events/customer.go

```diff
@@ -8,3 +12,14 @@ type CustomerAcquired struct {
 	AccountID    string
 	AccountName  string
 }
+
+// MessageDescription returns a human-readable description of the message.
+func (m CustomerAcquired) MessageDescription() string {
+	return fmt.Sprintf(
+		"acquired customer %s %s with first account %s %s",
+		m.CustomerID,
+		m.CustomerName,
+		m.AccountID,
+		m.AccountName,
```

Comment:
```suggestion
	return fmt.Sprintf(
		"acquired customer %s %s,
		m.CustomerID,
		m.CustomerName,
```

I can't think of a good way to word this one, but I think it's because the account details are kind of just "extra information", not absolutely germane to the event that occurred. 

We don't *have* to include every piece of information in the description, so maybe its best to remove this. What do you think?

Reply:
Yeah that sounds fine. The account opening will be included in a separate command+event.